doc/release-notes.md

```diff
@@ -1,3 +1,85 @@
+0.21.0+taproot0.1 Release Notes
+=========================
+
+Bitcoin Core version 0.21.0-based Taproot Client 0.1 is now available from:
```

Comment:
Maybe drop "version" or move it before 0.1?

Reply:
ACK